gradoop-common/src/main/java/org/gradoop/common/model/impl/properties/PropertyValueList.java

```diff
@@ -186,13 +187,14 @@ public boolean hasNext() {
     }
 
     @Override
-    public PropertyValue next() {
+    public PropertyValue next() throws NoSuchElementException {
       PropertyValue nextValue = new PropertyValue();
       try {
         DataInputView inputView = new DataInputViewStreamWrapper(inputStream);
         nextValue.read(inputView);
       } catch (IOException e) {
         e.printStackTrace();
```

Comment:
In the light of #1244 , maybe its a good idea to remove that one as well?

Reply:
YES